components/compiler-families/arm-compilers-devel/SPECS/arm-compilers-devel.spec

```diff
@@ -24,17 +25,25 @@ BuildArch: aarch64
 #!BuildIgnore: post-build-checks
 
 Requires: lmod%{PROJ_DELIM}
+Provides: %{pname}%{PROJ_DELIM}
```

Comment:
Provides: %{pname}%{PROJ_DELIM} should not be necessary to be listed explicitly.